scrapy/loader/__init__.py

```diff
@@ -3,219 +3,35 @@
 
 See documentation in docs/topics/loaders.rst
 """
-from collections import defaultdict
-from contextlib import suppress
+import itemloaders
 
 from scrapy.item import Item
-from scrapy.loader.common import wrap_loader_context
-from scrapy.loader.processors import Identity
 from scrapy.selector import Selector
-from scrapy.utils.misc import arg_to_iter, extract_regex
-from scrapy.utils.python import flatten
 
-
-def unbound_method(method):
-    """
-    Allow to use single-argument functions as input or output processors
-    (no need to define an unused first 'self' argument)
-    """
-    with suppress(AttributeError):
-        if '.' not in method.__qualname__:
-            return method.__func__
-    return method
-
-
-class ItemLoader:
+class ItemLoader(itemloaders.ItemLoader):
 
     default_item_class = Item
-    default_input_processor = Identity()
-    default_output_processor = Identity()
     default_selector_class = Selector
 
-    def __init__(self, item=None, selector=None, response=None, parent=None, **context):
+    def __init__(self, response=None, item=None, selector=None, parent=None, **context):
```

Comment:
Any reason for changing arguments order here? It might break code that doesn't make use of named args.